rust/ql/lib/codeql/rust/dataflow/internal/ModelsAsData.qll

```diff
@@ -0,0 +1,98 @@
+/**
+ * Defines extensible predicates for contributing library models from data extensions.
```

Comment:
In Swift we had a wonderful doc comment at the top of [`ExternalFlow.qll`](https://github.com/github/codeql/blob/main/swift/ql/lib/codeql/swift/dataflow/ExternalFlow.qll) that I regularly referred to.  It explaining the MAD format for Swift quite precisely including how to write sources, summaries and sinks; how to encode arguments, return values and so forth, and ideally it would have explained the different types of content as well.  I feel the same would be very valuable for Rust.

Reply:
I have added some QL doc.